.vscode/launch.json

```diff
@@ -20,6 +20,27 @@
 				"TARGET_BRANCH_NAME": "main",
 			},
 		},
+		{
+			"name": "codeCoverageAnalysis",
+			"program": "${workspaceFolder}/build-tools/packages/build-cli/bin/run.js",
+			"cwd": "${workspaceFolder}",
+			"request": "launch",
+			"skipFiles": ["<node_internals>/**"],
+			"type": "node",
+			"args": ["report", "codeCoverage"],
+			"env": {
+				// Your ADO PAT (Personal Access Token) User settings > Personal Access Tokens > New Token (check read permissions, likely Build, Release, Packaging, and Test Management)
+				"ADO_API_TOKEN": "",
+				"GITHUB_API_TOKEN": "",
+				"GITHUB_PR_NUMBER": "",
+				"ADO_BUILD_ID": "",
+				"ADO_CI_BUILD_DEFINITION_ID_BASELINE": "48",
+				"ADO_CI_BUILD_DEFINITION_ID_PR": "11",
+				"TARGET_BRANCH_NAME": "main",
+				"GITHUB_REPOSITORY_OWNER": "microsoft",
+				"GITHUB_REPOSITORY_NAME": "FluidFramework",
```

Comment:
While you can use the env variables for this, for local testing it's typically easier to pass at least some of them as flags instead of env variables.

Reply:
So, the oclif will collect from both whether we send them as flags or env variables?

Reply:
And easier in what sense? Since we are passing them like this in yml pipeline, we used it like this here.